### ISSUE TITLE
Displaying contours on shake maps

### ISSUE THREAD

**Issue by Charlotte-Morgan** [inasafe/inasafe-realtime#152]
Hadi recommend that when MMI contours are displayed on maps they can be solid lines over land and dashed lines over water.
The reason is that the contours are for MMI which is shaking on land. There is no such think as MMI for water :)
cc
@hadighasemi

**Comment by lucernae** [comment]
I remember he mentioned this one too.
@ismailsunni can you help me investigate how to do that? I think this is not straightforwad. The MMI contours will need to be clipped with polygons which have land/water columns. Then we have to put attribute for line, so we can style it based on land/water attribute. Or do you have better idea?

**Comment by ismailsunni** [comment]
@lucernae I think that's also my idea

**Comment by Charlotte-Morgan** [comment]
This becomes a cartographic solution - what data will you share for people to download?

**Comment by gubuntu** [comment]
@ivanbusthomi see how far you can get with QGIS styling

**Comment by ivanbusthomi** [comment]
> ivanbusthomi see how far you can get with QGIS styling

Hi @gubuntu - I can't quite remember whether we already made a decision regarding this issue, but I just managed to find a good solution for this.

This is what I can get at the moment using the geometry generator option in layer styling
![image](https://user-images.githubusercontent.com/7351977/40406940-d3a6b74e-5e8c-11e8-9720-fb3ad87cf984.png)

We only need to load a new polygon layer that indicates the land (country boundary). The geometry generator option can be saved into the qml, which then can be used to specify the contour style.

I think the only drawback is that it took several seconds to render it.

@Charlotte-Morgan @timlinux @ismailsunni @lucernae
